login/packets.go

```diff
@@ -33,6 +33,32 @@ func packetLoginResponce(result byte, userID int32, gender byte, isAdmin bool, u
 	return pac
 }
 
+func packetPinOperation(mode byte) mpacket.Packet {
+	pac := mpacket.CreateWithOpcode(opcode.SendLoginPinRegister)
```

Comment:
I think a better name for `SendLoginPinRegister` would be `SendLoginPinOperation` as this opcode is used for more than registering.